relay-pii/src/eap.rs

```diff
@@ -0,0 +1,2562 @@
+use relay_event_schema::processor::{
+    ProcessValue, ProcessingResult, ProcessingState, ValueType, process_value,
+};
+use relay_protocol::Annotated;
+
+use crate::{AttributeMode, PiiConfig, PiiProcessor};
+
+/// Scrubs an EAP item (such as an `OurLog` or `SpanV2`) with the given PII configs.
+pub fn scrub_eap_item<T: ProcessValue>(
```

Comment:
Maybe a bit redundant as you are already in an `eap` module. Usage wise I think `eap::scrub()` would work well.